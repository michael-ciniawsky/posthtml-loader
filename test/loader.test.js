'use strict'

const webpack = require('./helpers/compiler')

describe('Loader', () => {
  test('Defaults', () => {
    const config = {
      loader: {
        test: /\.html$/,
        options: {}
      }
    }

    return webpack('fixture.js', config)
      .then((stats) => {
        const [module] = stats.toJson().modules

        expect(module.source).toMatchSnapshot()
      })
      .catch((err) => err)
  })
})
