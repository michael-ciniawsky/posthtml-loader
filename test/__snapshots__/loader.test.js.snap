// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Defaults 1`] = `
"export default \`<section>Hello</section>
\`"
`;
